.github/workflows/test.yml

```diff
@@ -28,7 +28,8 @@ jobs:
           - '3.9'
           - '3.10'
           - '3.11'
-          - 'pypy-3.7'
+          - 'pypy-3.8'
+          - 'pypy-3.9'
```

Comment:
not sure if related but I also asked bot to renew https://github.com/conda-forge/datalad-feedstock/pull/101